client/containers/Pub/PubSubmissionHeader/spubHeader.scss

```diff
@@ -0,0 +1,43 @@
+.spub-header-component {
+	border-bottom:	border-bottom: 1px solid #d2d2d2;
+;
+	padding: 200px;
+
+	h1 {
```

Comment:
We might want to give this a more specific class name, like `submission-workflow-title`. This will differentiate it from any `h1` elements in the user-provided `instructionsText`.

Reply:
Oh, one more. I think we lost the `$header-font` here.

Reply:
How would i use the classname for a DocJSON